kube-client/src/config/file_loader.rs

```diff
@@ -59,40 +59,42 @@ impl ConfigLoader {
         cluster: Option<&String>,
         user: Option<&String>,
     ) -> Result<Self, KubeconfigError> {
+        let current_context_name = &config.current_context.clone().unwrap_or_default();
         let context_name = if let Some(name) = context {
             name
-        } else if let Some(name) = &config.current_context {
-            name
         } else {
-            return Err(KubeconfigError::CurrentContextNotSet);
+            current_context_name
         };
+
         let current_context = config
             .contexts
             .iter()
-            .find(|named_context| &named_context.name == context_name)
-            .map(|named_context| &named_context.context)
+            .find(|named_context| &named_context.name.clone().unwrap_or_default() == context_name)
```

Comment:
```suggestion
            .find(|named_context| named_context.name.as_ref() == Some(context_name))
```

Reply:
It's related to the comment above, if I apply this suggested change, then we it won't work when context name is null/none

Reply:
Resolving this, no longer relevant after reverting.